### PR TITLE
chore(): Switch from travis to githubaction

### PR DESCRIPTION
**Pull request by nbrouand**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -94,8 +115,8 @@ publishing {
                     url.set("${gitHubUrl}/issues")
                 }
                 ciManagement {
-                    system.set("travis-ci")
-                    url.set("https://travis-ci.org/chutney-testing/${project.name}")
+                    system.set("github-ci")
+                    url.set("https://github.com/chutney-testing/chutney-kotlin-dsl/actions")
```
Comment (boddissattva) [inline build.gradle.kts:119]: Use ${gitHubUrl}

File: build.gradle.kts [hunk 2]
```diff
@@ -5,18 +5,35 @@ import java.time.format.DateTimeFormatter
 plugins {
     `maven-publish`
     kotlin("jvm") version "1.4.0"
+    id("com.jfrog.bintray") version "1.8.5"
     id("org.ajoberstar.reckon") version "0.12.0"
 }
 
+val group = "com.chutneytesting"
+val projectName = "chutney-kotlin-dsl"
```
Comment (boddissattva) [inline build.gradle.kts:13]: Is it necessary to redefine default value ?

File: build.gradle.kts [hunk 3]
```diff
@@ -71,8 +89,11 @@ tasks {
 
 publishing {
     publications {
-        create<MavenPublication>("default") {
+        create<MavenPublication>(publicationName) {
             from(components["java"])
+            groupId = group
+            artifactId = projectName
+            version = project.version.toString()
```
Comment (boddissattva) [inline build.gradle.kts:96]: Is it necessary to redefine default values ?

File: build.gradle.kts [hunk 4]
```diff
@@ -5,18 +5,35 @@ import java.time.format.DateTimeFormatter
...
-group = "com.chutneytesting"
-
-val timestamp: String = DateTimeFormatter.ISO_INSTANT.format(Instant.now())
-val gitHubUrl = "https://github.com/chutney-testing/${project.name}"
+bintray {
+    user = System.getenv("BINTRAY_SERVER_USERNAME")
+    key = System.getenv("BINTRAY_SERVER_PASSWORD")
+    publish = true
+    setPublications(publicationName)
+    pkg(delegateClosureOf<com.jfrog.bintray.gradle.BintrayExtension.PackageConfig> {
```
Comment (boddissattva) [inline build.gradle.kts:28]: Here https://docs.gradle.org/current/userguide/kotlin_dsl.html#groovy_closures_from_kotlin, closureOf() is used. Have you experienced NPE to use the delegateClosureOf() method ?

**Review by boddissattva — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 3 above)
Comment (nbrouand) [reply build.gradle.kts:95]: mandatory from bintray plugin

**Review by boddissattva — APPROVED** [review]

**Pull request merged**